e2e-tests/cypress/e2e/journeys/insurance/policy-and-exports/single-contract-policy/single-contract-policy.spec.js

```diff
@@ -51,7 +51,7 @@ const {
         CREDIT_PERIOD_WITH_BUYER,
```

Comment:
getDate import is not used.  Does not let me comment there (line 3)

Reply:
Sorry forgot to push, fixed